src/deluge/gui/ui/slicer.cpp

```diff
@@ -649,7 +650,7 @@ void Slicer::doSlice() {
 		uint32_t nextDrumStart = lengthInSamples / numClips;
 		firstRange->sampleHolder.endPos = nextDrumStart;
 
-		firstDrum->sources[0].repeatMode = (lengthMSPerSlice < 2002) ? SampleRepeatMode::ONCE : SampleRepeatMode::CUT;
+		firstDrum->sources[0].repeatMode = (lengthMSPerSlice < 2002) ? SampleRepeatMode::ONCE : defaultSliceMode;
```

Comment:
defaultSliceMode should be FlashStorage::defaultSliceMode

Reply:
yes, should be

Reply:
well I did everything but it still says these Operands have a different type and then it aborts. hm, why is that?